src/NServiceBus.Core/Unicast/Config/UnicastBus.cs

```diff
@@ -34,44 +28,6 @@ static EndpointInstance GetEndpointInstanceName(ReadOnlySettings settings)
         protected internal override void Setup(FeatureConfigurationContext context)
         {
             context.Container.ConfigureComponent<BusNotifications>(DependencyLifecycle.SingleInstance);
-
```

Comment:
Wow. Cool that stuff that belongs to serializer has been inlined in the serializer feature